hack/aks/Makefile

```diff
@@ -99,98 +99,97 @@ overlay-byocni-up: rg-up overlay-net-up ## Brings up an Overlay BYO CNI cluster
 		--yes
 	@$(MAKE) set-kubeconf
 
-overlay-cilium-up: rg-up overlay-net-up ## Brings up an Overlay Cilium cluster
+overlay-byocni-nokubeproxy-up: rg-up overlay-net-up ## Brings up an Overlay BYO CNI cluster without kube-proxy for Cilium
```

Comment:
the comments says `cilium` but the cmd is not using cilium ? The usage is for `cilium` but that can be for any CNI right ?